### PR TITLE
DEID-2625 Added Image Options to PAI Client

### PR DESCRIPTION
**Pull request by AmirPAI**
### What's Changed?

- Added ImageOptions class
- Added to Base64 route
- Added to URI route
- Added 8 Unit Tests for request object of ImageOptions 
- Added Integration test

### PR Checklist

- [x] Updated unit tests
- [x] Ran unit tests

### CODE REVIEW

**Review by a-guiducci — APPROVED** [review]
Looks good! Can you update the appropriate CHANGELOG section before merging please :)

File: src/privateai_client/components/request_objects.py [hunk 1]
```diff
@@ -852,12 +911,14 @@ def fromdict(cls, values: dict):
...
                     initializer_dict[key] = AudioOptions.fromdict(value)
+                elif key == "image_options":
+                    initializer_dict[key] = ImageOptions.fromdict(value)
                 else:
                     initializer_dict[key] = value
             return cls._fromdict(initializer_dict)
         except TypeError:
             raise TypeError(
-                "ProcessFileBase64Request can only accept the values 'file', 'entity_detection', 'pdf_options and 'audio_options'"
+                "ProcessFileBase64Request can only accept the values 'file', 'entity_detection', 'pdf_options', 'audio_options' and 'image_options'"
```
Comment (letmerecall) [inline src/privateai_client/components/request_objects.py:921]: Nitpick

```suggestion
                "ProcessFileBase64Request can only accept the values 'file', 'entity_detection', 'pdf_options', 'audio_options', and 'image_options'"
```

File: src/privateai_client/tests/test_request_objects.py [hunk 2]
```diff
@@ -969,7 +1032,7 @@ def test_process_file_uri_request_initialize_fromdict():
 
 def test_process_file_uri_request_invalid_initialize_fromdict():
     error_msg = (
-        "ProcessFileUriRequest can only accept the values 'uri', 'entity_detection', 'pdf_options and 'audio_options'"
+        "ProcessFileUriRequest can only accept the values 'uri', 'entity_detection', 'pdf_options', 'audio_options' and 'image_options'"
```
Comment (letmerecall) [inline src/privateai_client/tests/test_request_objects.py:1035]: Nitpick

```suggestion
        "ProcessFileUriRequest can only accept the values 'uri', 'entity_detection', 'pdf_options', 'audio_options', and 'image_options'"
```

File: src/privateai_client/tests/test_request_objects.py [hunk 3]
```diff
@@ -1082,7 +1145,7 @@ def test_process_file_base64_request_initialize_fromdict():
 
 
 def test_process_file_base64_request_invalid_initialize_fromdict():
-    error_msg = "ProcessFileBase64Request can only accept the values 'file', 'entity_detection', 'pdf_options and 'audio_options'"
+    error_msg = "ProcessFileBase64Request can only accept the values 'file', 'entity_detection', 'pdf_options', 'audio_options' and 'image_options'"
```
Comment (letmerecall) [inline src/privateai_client/tests/test_request_objects.py:1148]: Nitpick

```suggestion
    error_msg = "ProcessFileBase64Request can only accept the values 'file', 'entity_detection', 'pdf_options', 'audio_options', and 'image_options'"
```

**Review by letmerecall — APPROVED** [review]
Nice one @AmirPAI, just some nitpicks. Otherwise LGTM.

**Comment by AmirPAI** [comment]
Thanks @a-guiducci @letmerecall for the review. Do check if the CHANGELOG update is good.

**Comment by a-guiducci** [comment]
Looks good, thanks @AmirPAI!

**Comment by letmerecall** [comment]
fixes #42

**Pull request merged**